File: scrape.py
import urllib.request
from html.parser import HTMLParser
import ssl

ctx = ssl.create_default_context()
ctx.check_hostname = False
ctx.verify_mode = ssl.CERT_NONE

class MyHTMLParser(HTMLParser):
    def __init__(self):
        super().__init__()
        self.images = []
    def handle_starttag(self, tag, attrs):
        if tag == "img":
            for attr in attrs:
                if attr[0] in ["src", "data-src"]:
                    self.images.append(attr[1])

req = urllib.request.Request("https://favercar.com/location-voiture-luxe/", headers={'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)'})
try:
    html = urllib.request.urlopen(req, context=ctx).read().decode('utf-8')
    parser = MyHTMLParser()
    parser.feed(html)
    for img in set(parser.images):
        if img.endswith(('.jpg', '.png', '.webp', '.jpeg')):
            print(img)
except Exception as e:
    print("Error:", e)

File: scrape.js
const https = require('https');

https.get('https://favercar.com/location-voiture-luxe/', (res) => {
    let data = '';
    res.on('data', (chunk) => data += chunk);
    res.on('end', () => {
        const regex = /<img[^>]+src="([^">]+)"/g;
        let match;
        const images = new Set();
        while ((match = regex.exec(data)) !== null) {
            if (match[1].match(/\.(jpeg|jpg|gif|png|webp)/i)) {
                images.add(match[1]);
            }
        }
        
        // Also look for data-src
        const regexDataSrc = /<img[^>]+data-src="([^">]+)"/g;
        while ((match = regexDataSrc.exec(data)) !== null) {
            if (match[1].match(/\.(jpeg|jpg|gif|png|webp)/i)) {
                images.add(match[1]);
            }
        }
        
        console.log(Array.from(images).join('\n'));
    });
}).on('error', (err) => {
    console.log("Error: " + err.message);
});
